google/cloud/aiplatform/experimental/vertex_model/base.py

```diff
@@ -81,15 +104,12 @@ def health():
 async def predict(request: Request):
     body = await request.json()
     instances = body["instances"]
+    input_data = original_model.predict_payload_to_predict_input(instances)
 
-    feature_columns = ['feat_1', 'feat_2']
-    data = pd.DataFrame(instances, columns=feature_columns)
-    torch_tensor = torch.tensor(data[feature_columns].values).type(torch.FloatTensor)
-
-    my_model.predict = functools.partial(original_model.__class__.predict, my_model)
-    outputs = my_model.predict(torch_tensor)
+    my_model.predict = wrapped_model.predict
+    outputs = my_model.predict(input_data)
 
-    return {"predictions": outputs.tolist()}
+    return {"predictions": original_model.predict_output_to_predict_payload(outputs)}
```

Comment:
Should this be `my_model`?

Reply:
See above comment -- the deserialized model won't contain these functions.